edge-patron/src/main/resources/vega/edge-patron/features/lc-user-registration.feature

```diff
@@ -0,0 +1,179 @@
+Feature: LC user registration tests tests
+
+  Background:
+    * url baseUrl
+    * callonce login { tenant: 'diku', name: 'diku_admin', password: 'admin' }
```

Comment:
Cant we use admin login instead of having hard dependency on diku ?

Reply:
fixed=> https://github.com/folio-org/folio-integration-tests/pull/1463/commits/79bfbb0114f2fc87f2f453d107b54e661b5ef0ac